api/db/changelog/db.changelog-master.xml

```diff
@@ -216,6 +216,7 @@
   <include file="changelog/db.changelog-206-demographic-survey-v2-colombian-fix.xml"/>
   <include file="changelog/db.changelog-207-cdr-version-hasFitbitSleep-hasSurveyConduct-field.xml"/>
   <include file="changelog/db.changelog-208-add-new-user-satisfaction-survey.xml"/>
+  <include file="changelog/db.changelog-changelog-209-drop-admin-action-history-table.xml"/>
```

Comment:
This appears to be a different file name (extra `-changelog`) than the changelog file.

Reply:
Good catch.